jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -613,11 +621,12 @@ def import_trace(self, trace):
         if isinstance(trace, tracing.FlatTrace):
             self.trace_type_selected = 'Flat'
             self.trace_pixel = trace.trace_pos
-        elif isinstance(trace, tracing.KosmosTrace):
-            self.trace_type_selected = 'Auto'
+        elif isinstance(trace, tracing.FitTrace):
+            self.trace_type_selected = trace.trace_model.__class__.__name__.strip('1D')
```

Comment:
Today I learned that the `str.strip` method takes arguments. Thanks for teaching me something! 😃